### PR TITLE
Deploy to radix

### PR DESCRIPTION
**Pull request by soofstad**
- Github Actions to build and push images to mariner.acr.io/for/*
- Github Action to deploy master branch to Radix
- radixconfig.yaml file

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,25 +5,27 @@ services:
...
     depends_on:
       - dmss
     environment:
       LOGGING_LEVEL: DEBUG
       ENVIRONMENT: local
-      DMSS_HOST: dmss
-      DMSS_PORT: 5000
+      # DMSS_HOST: dmss-forecast-of-response-test.radix.equinor.com
+      # DMSS_PORT: 80
+     DMSS_HOST: dmss
```
Comment (netr0m) [inline docker-compose.yml:17]: Missing a couple of spaces here, no?

Comment (netr0m) [reply docker-compose.yml:17]: #L17 &&  #L18

File: docker-compose.yml [hunk 2]
```diff
@@ -5,25 +5,27 @@ services:
     build:
       context: ./api
       args:
-        DMT_VERSION: "${DMT_VERSION}"
+        DMT_VERSION:-latest: ${DMT_VERSION:-latest}
     restart: unless-stopped
     depends_on:
       - dmss
     environment:
       LOGGING_LEVEL: DEBUG
```
Comment (netr0m) [inline docker-compose.yml:13]: TODO?: Disable `DEBUG` in "prod"? At least for later

**Review by netr0m — APPROVED** [review]
Looks good! I left a couple of remarks for consideration

File: docker-compose.yml (hunk 1 above)
Comment (soofstad) [reply docker-compose.yml:17]: fixed

**Pull request merged**